lib/middleware/session.js

```diff
@@ -79,7 +79,14 @@ var ServletSession = exports.ServletSession = function(request) {
                 // to the attributes in the servlet session object.
                 data = new JavaAdapter(org.mozilla.javascript.NativeObject, {
                     put: function(name, start, value) {
-                        getSession().setAttribute(name, value);
+                    	if (typeof(value) === 'undefined' || value === null) {
+                    		getSession().removeAttribute(name);
+                    		if (Object.prototype[name]) {
+                                Object.prototype[name] = value;
```

Comment:
I don't have use for them... I'll remove lines 84-86 then. I'm wondering if someone else may be relying on the lines in the get function? Do you think that might mess someone's app upon upgrade to the latest ringo?